Adoneg/ex6/Q4.py

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
your calculator does not meet the  conditions which the question was set. I said that the user should be able to perform any number of operations eg 1 + 2 * 4 / 5 , but your code doesn't do that. Please review your code